lib/gen/api/enums/v1/command_type_pb.rb

```diff
@@ -0,0 +1,35 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
I'd say lets avoid checking in generated code unless we have a really compelling reason to do so

Reply:
Disagree :-)

Reply:
In some languages with better code-gen/build-script stories or in languages where it's expected that the code in the repo doesn't match the code published to the package manager it can _kinda_ be ok. But for others where 1) people expect code in repo to be what's published, 2) there is no benefit to making people generate protos to build the project, 3) often documentation likes to link to actual-repo code, and/or 4) doesn't have great build script support, committing this code makes sense.

We ended up having to commit it in Python when the gRPC generator stopped working for M1users and we realized burdening users with generating proto code for no real benefit wasn't worth it. Of course I made a CI build step that also does the code gen and confirms that the files in the PR match what the code-gen would produce (basically a code-gen + git diff). That way we know there's never a staleness concern.